OMCompiler/Compiler/BackEnd/Initialization.mo

```diff
@@ -917,11 +917,7 @@ algorithm
     Error.addSourceMessage(Error.UNBOUND_PARAMETER_WITH_START_VALUE_WARNING, {s, str}, info);
```

Comment:
The line above look like they would add the default start value, but it is not used in SimCode. An enum parameter without start value is initialized as an integer with a zero value.
Anyone got an idea why this happens?